thorlcr/activities/keyedjoin/thkeyedjoinslave-legacy.cpp

```diff
@@ -1038,7 +1038,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                     pendingSendsSem.signal();
                 }
                 crit.leave();
-                threaded.join();
+                threaded.join(INFINITE, false);
```

Comment:
I don't think this one is necessary, i.e. ok to throw exception here in CKeyedFetchHandler::stop()

Reply:
Ok, but stop() is called from the destructor I think

Reply:
@richardkchapman - This stop() isn't. Destructor in question is ~CKeyedFetchHandler()

Reply:
Ok, fixed